plasmapy/physics/transport/tests/test_transport.py

```diff
@@ -7,41 +7,30 @@
 from astropy import units as u
 from plasmapy.atomic.atomic import ion_mass, integer_charge
 from plasmapy.utils.exceptions import (PhysicsError, PhysicsWarning,
-                                       RelativityWarning, RelativityError)
+                                       RelativityWarning, RelativityError,
+                                       InvalidParticleError)
 from plasmapy.physics.parameters import Hall_parameter
-from ...constants import c, m_p, m_e, e, mu0
-
-from ..transport import (Coulomb_logarithm,
-                         b_perp,
-                         impact_parameter,
-                         collision_frequency,
-                         mean_free_path,
-                         Spitzer_resistivity,
-                         mobility,
-                         Knudsen_number,
-                         coupling_parameter,
-                         classical_transport,
-                         _nondim_tc_e_braginskii,
-                         _nondim_tc_i_braginskii,
-                         _nondim_tec_braginskii,
-                         _nondim_resist_braginskii,
-                         _nondim_visc_i_braginskii,
-                         _nondim_visc_e_braginskii,
-                         _nondim_tc_e_spitzer,
-                         _nondim_tec_spitzer,
-                         _nondim_resist_spitzer,
-                         _nondim_thermal_conductivity,
-                         _nondim_resistivity,
-                         _nondim_te_conductivity,
-                         _nondim_tc_e_ji_held,
-                         _nondim_tc_i_ji_held,
-                         _nondim_tec_ji_held,
-                         _nondim_resist_ji_held,
-                         _nondim_visc_e_ji_held,
-                         _nondim_visc_i_ji_held,
-                         _nondim_viscosity,
-                         _check_Z,
-                         )
+from plasmapy.constants import c, m_p, m_e, e, mu0
+
+from plasmapy.physics.transport.collisions import (Coulomb_logarithm,
+                                                   b_perp,
+                                                   impact_parameter,
+                                                   collision_frequency,
+                                                   mean_free_path,
+                                                   Spitzer_resistivity,
+                                                   mobility,
+                                                   Knudsen_number,
+                                                   coupling_parameter,
+                                                   )
+from plasmapy.physics.transport.braginskii import _nondim_thermal_conductivity, \
```

Comment:
kill all the continuation slashes :see_no_evil: 

Seriously, this should be in parentheses.

Reply:
Completely forgot!